src/util/graasp-error.ts

```diff
@@ -316,6 +316,20 @@ export class OrphanSession extends BaseGraaspError {
   }
 }
 
+export class InvalidPassword extends BaseGraaspError {
+  constructor(data?: unknown) {
+    // this status code is custom for the browser to know it needs to refresh its token
+    super(
+      {
+        code: 'GERR025',
+        statusCode: StatusCodes.UNAUTHORIZED,
+        message: 'Current password is invalid. Please try again.',
```

Comment:
Could you create a branch in translations as well? Or an issue? So we can later translate the errors.